packages/web/src/components/RedwoodApolloProvider.tsx

```diff
@@ -20,12 +23,33 @@ const ApolloProviderWithFetchConfig: React.FunctionComponent<{
   config?: Omit<ApolloClientOptions<InMemoryCache>, 'cache'>
 }> = ({ config = {}, children }) => {
   const { uri, headers } = useFetchConfig()
```

Comment:
We could remove `headers` here since this is where the auth-token and type came from previously.

Reply:
Doing so would save you an extra re-render.

Reply:
Yeah I intentionally didn't do that, want to remove this if/when FetchContext is removed if it makes sense?

Reply:
Yeah that make sense.